lms/djangoapps/courseware/grades.py

```diff
@@ -267,6 +268,8 @@ def create_module(descriptor):
 
         totaled_scores[section_format] = format_scores
 
+    # Grading policy might be overriden by a POC, need to reset it
+    course.set_grading_policy(course.grading_policy)
```

Comment:
We'd have to get @lou-wang and @explorerleslie to chime in for the official edX stance on it, but perhaps we could just add some clear messaging and code that skips POC students during the cert generation process?